demo/index.css

```diff
@@ -1,46 +1,579 @@
-﻿[draggable] {
-  user-select: none;
+﻿* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Poppins, sans-serif;
+    font-weight: 400;
+}
+
+html,
+body {
+    position: relative;
+    width: 100%;
+    height: 100dvh;
+    overflow: hidden;
+}
```

Comment:
Let's not do this: this is a normal web page, not a single-page app. The polyfill should work for pages with scrollbars just fine.

Reply:
Also, since CSS nesting has been part of the standard for a year now, and tooling like VS Code understand nesting just fine, let's not add a whole bunch of flat rules. Nest where we can, and if we need "component"-specific rules that need global declaration, we can `@import` those from a separate file (although I don't think that necessary here)